vignettes/tidynhl.Rmd

```diff
@@ -157,5 +157,5 @@ playoff year while he was anywhere but in the Sunshine State.
 
 ## Conclusion
 
-As we saw by the reading of this short introduction to `tidynhl`, the main goal of the package is to make it easy to retrieve NHL data, even for R beginning
+As we saw by in this short introduction to `tidynhl`, the main goal of the package is to make it easy to retrieve NHL data, even for R beginning
```

Comment:
"As we saw by in" should be "As we saw in" I guess... Thanks to correct and commit, I'll then merge the PR